ts/session/apis/snode_api/swarmPolling.ts

```diff
@@ -265,9 +266,16 @@ export class SwarmPolling {
     // check if we just fetched the details from the config namespaces.
     // If yes, merge them together and exclude them from the rest of the messages.
     if (userConfigLibsession && resultsFromAllNamespaces) {
-      const userConfigMessages = resultsFromAllNamespaces
-        .filter(m => SnodeNamespace.isUserConfigNamespace(m.namespace))
-        .map(r => r.messages.messages);
+      const userConfigMessages = resultsFromAllNamespaces.filter(m =>
+        SnodeNamespace.isUserConfigNamespace(m.namespace)
+      );
+
+      const userConfigMessagesWithNamespace: Array<Array<RetrieveMessageItemWithNamespace>> =
+        userConfigMessages.map(r => {
+          return (r.messages.messages || []).map(m => {
```

Comment:
lol

Reply:
yeah that sucks, see my discussion in the chat